api/v1/app/models.py

```diff
@@ -0,0 +1,138 @@
+import datetime
+import jwt
+from werkzeug.security import generate_password_hash, check_password_hash
+from flask import current_app
+
+
+users=[]
+
+class User():
+    primary_key = 0
+    db_users = []
+
+    def __init__(self, username, email, address, password):
+        self.id = User.primary_key
+        self.username = username
+        self.email = email
+        self.address = address
+        self.password = generate_password_hash(password)
+        User.primary_key +=1
+
+    def save(self):
+        user_detail = {}
+        user_detail['id'] = User.primary_key
+        user_detail['username'] = self.username
+        user_detail['email'] = self.email
+        user_detail['address'] = self.address
+        user_detail['password'] = self.password
+        User.db_users.append(user_detail)
```

Comment:
You could populate the dictionary when creating it. No need to type `user_detail['key']` for each key.
```
def save(self)
    user_detail = {
      'id': User.primary_key
      'username': self.username
      'email': self.email
      'address': self.address
      'password': self.password
    }
    User.db_users.append(user_detail)
```

Reply:
I have a quiz @JoshuaOndieki,  how is popuplating when creating differ with how he did, or what is the advantage... if you know of any. I like to understand stuffs

Reply:
Not sure if there are any performance advantages.
But for me, I would say it improves readability.
P.S. *My opinion*

Reply:
Thanks for your feedbacks but i don't think its a good idea to populate the dictionary directly.i prefer using a function that does that for me instead of calling the data structure every time i want to populate it.
Plus i might run into instance missing error when i try to access the Object without an instance